datamade/la-metro-councilmatic
# Add autocomplete search with SmartLogic's Semantic Enhancement Search API

## Overview

This PR:

- Bundles [Devbridge's jQuery autocomplete library](https://github.com/devbridge/jQuery-Autocomplete).
- Implements a Django view to retrieve a JSON web tokens for SmartLogic's Semantic Enhancement Search API.
- Implements autocomplete functionality drawing suggestions from the SES API on the home page and search results page.

### Checklist


## Testing Instructions

- If needed, regenerate the Solr Schema
- Update your `settings_deployment` with the SmartLogic API keys
- In one terminal window `docker-compose up solr-staging`, in another run the server
- Begin typing a topic, e.g. Metro Green Line
- Confirm autocomplete appears and is functional
- Choose a topic, confirm it goes to the appropriate search page when selected

Handles #433
